static/css/restyle.less

```diff
@@ -256,10 +275,10 @@ blue-pattern, blue-gradient, white overlay
 
 /* Top Menu */
 /*seperator */
-#aux-nav > ul > li:not(:first-child):before {
-    ul {
-        li {
-            /* Other Applications Panel trinagle position*/
+#aux-nav {
+    > ul {
```

Comment:
Good call on both.

Because there are a lot of overrides happening I’m not sure, but I’ll experiment and clean it up as much as possible.